translations/Readme.md

```diff
@@ -0,0 +1,138 @@
+# About Translation
+
+## Files list
+
+- [Windows](#windows)
+
+    ````text
+    lupdate.cmd
+    lrelease.cmd
+    ````
+
+- [macOS / Linux](#macos-linux)
+
+    ```` text
+    translation_utility.sh
+    ````
+
+## How to use the translation utilities
+
+### Requirement
+
+- These utilities needs [QT](https://qt.io) library.
+
+### On Windows {#windows}
+
+#### 1. Adding translation `(.ts)` files
+
+1. Open `lupdate.cmd` in editor. (e.g, VSCode or Notepad, etc.)
+2. Append your new file name to the end of the first line, in the format **locale_**_language-code_**.ts** (e.g, locale_de, local_zh-cn, etc.).
+3. Double click on `lupdate.cmd` or run it in `cmd.exe`.
+4. You will find the new file in the same directory as the `lupdate.cmd`.
+
+#### 2. Updating `.qm` files
```

Comment:
You swapped step 2 and 3, translation needs to go before lrelease (and after lupdate)